### DISCUSSION TITLE
Blessed be lllyasviel

### DISCUSSION THREAD

**Discussion by ali0une** [lllyasviel/stable-diffusion-webui-forge#1061] [Show and tell]
Everybody please hail the King!

>dragon ball hero holding a sign with text 'Blessed be lllyasviel' \<lora:flux1_art_v1.0:1\> art \<lora:flux1_anime_v1.0:1\> anime
Steps: 20, Sampler: Euler, Schedule type: Simple, CFG scale: 1, Distilled CFG Scale: 3.5, Seed: 1120836892, Size: 768x1344, Model hash: c161224931, Model: flux1-dev-bnb-nf4, RNG: CPU, Lora hashes: "flux1_art_v1.0: 08da7f170659, flux1_anime_v1.0: f8080ae541ca", Version: f2.0.1v1.10.1-previous-252-g00f1cd36, Module 1: flux1-dev-t5xxl_fp16, Module 2: flux1-dev-vae-float16, Module 3: flux1-dev-clip_l
Time taken: 24.9 sec.

![image](https://github.com/user-attachments/assets/1d9f3fff-7a9f-48aa-a91a-d8557825ad7c)

**Comment by queenofinvidia** [comment]
illya is the goat!

flux loras work with nf4?

**Reply by ali0une** [reply]
Not yet https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/1038#discussioncomment-10328760

**Reply by abzaloff** [reply]
> flux loras work with nf4?

Yes, it works. It is necessary to select the mode: Automatic (fp16 LoRA)

**Comment by BNP1111** [comment]
goat!goat!goat!

**Reply by abzaloff** [reply]
> goat!goat!goat!

Turn away from the mirror already

**Comment by ZeroCool22** [comment]
GPU?

**Reply by ZeroCool1977** [reply]
> GPU?

aha! finally, i have found you namesake!  totally off topic & completely insignificant, but this character name ( Zero Cool) from the film "Hackers" is used almost everywhere when i try to start an account (despite it being fairly old & basically Angelina Jolie's first film), yet i have never spotted it in use until now.
& Illyasviel rocks!!! so grateful for forge & appreciate all the work & effort. it's nice to see both names (Illyasviel & Forge) suddenly popping up & getting recognition across the internet. i used to get the sound of crickets when i told ppl i used forge, but certainly not any more!
